### PR TITLE
add Header and Footer component

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add the code of Header and Footer component in the development branch

In this PR, **427 additions and 45 deletions** lines of code.

If you don't like it, you can ignore to review the following file
- package
- image
- manifest

I have also added comment in the code, where it is difficult to understand the purpose.

### Please review this PR, and give your feedback

**Folder Structure**
- image
- image
- image
- image

**This is how, it looks**
image

### CODE REVIEW

**Review by b0llu — APPROVED** [review]
Nothing to suggest. Looks good

File: src/components/header/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import { NavbarLogInBtn, NavbarInputForm } from "./shared";
+
+/**
+ * NavbarBadgeLink - component for badge link
+ */
+const NavbarBadgeLink = ({ badgeIconType, className }) => {
+  const [url, linkFor] =
+    badgeIconType === "heart"
+      ? [ROUTE_WISHLIST, "My WishLists"]
+      : [ROUTE_CART, "My Cart"];
```
Comment (foolhardy21) [inline src/components/header/Navbar.jsx:16]: Appreciation: good use of array destructuring.

File: src/utils/constants/routes.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const ROUTE_HOME = "/home";
```
Comment (foolhardy21) [inline src/utils/constants/routes.js:1]: Appreciation: use of constant variables.

**Review by foolhardy21 — APPROVED** [review]
good to go ahead.

**Comment by hsnice16** [comment]
Thank you for reviewing.

**Pull request merged**